api-reference/50 Common/Object Structures/PdfCell/wordWrapEnabled.md

```diff
@@ -0,0 +1,10 @@
+---
+id: PdfCell.wordWrapEnabled
+type: Boolean
+---
+---
+##### shortDescription
+Specifies whether to enable word wrapping for exported data in the resulting PDF file.
+
+---
+The default value depends on the, the value of the grid's [wordWrapEnabled](/api-reference/10%20UI%20Components/GridBase/1%20Configuration/wordWrapEnabled.md '/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#wordWrapEnabled') option.
```

Comment:
The default value depends on **the** value of the grid's [wordWrapEnabled](/api-reference/10%20UI%20Components/GridBase/1%20Configuration/wordWrapEnabled.md '/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#wordWrapEnabled') option.